soc/realtek/ec/rts5912/reg/reg_rtmr.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * @brief RTOS Timer Controller (RTMR)
+ */
+
+typedef struct {
+	volatile uint32_t  LDCNT;
+	volatile uint32_t  CNT;
+
+	union {
+		volatile uint32_t CTRL;
+
+		struct {
+			volatile uint32_t EN		: 1;
+			volatile uint32_t MDSEL		: 1;
+			volatile uint32_t INTEN		: 1;
+			uint32_t reserved1			: 29;
```

Comment:
indentation looks off, is it just the github view?

Reply:
We run clang-format in new commit.
Now it looks like this.
			volatile uint32_t EN: 1;
			volatile uint32_t MDSEL: 1;
			volatile uint32_t INTEN: 1;
			uint32_t reserved1: 29;